coldfront/config/plugins/ldap_user_search.py

```diff
@@ -14,4 +14,6 @@
 LDAP_USER_SEARCH_BASE = ENV.str('LDAP_USER_SEARCH_BASE')
 LDAP_USER_SEARCH_BIND_DN = ENV.str('LDAP_USER_SEARCH_BIND_DN')
 LDAP_USER_SEARCH_BIND_PASSWORD = ENV.str('LDAP_USER_SEARCH_BIND_PASSWORD')
+LDAP_USER_SEARCH_CONNECT_TIMEOUT = ENV.int('LDAP_USER_SEARCH_CONNECT_TIMEOUT', default=1)
```

Comment:
Looks like the default connect_timeout in ldap3 is set to None which ends up using a default value of 2.5. See [here](https://github.com/cannatag/ldap3/blob/dev/ldap3/utils/config.py#L82). Perhaps we should use a similar default, so either set to 2.5 or even None?

Reply:
Agreed 2.5 is probably more reasonable.